recipes/micromamba-server/meta.yaml

```
@@ -0,0 +1,94 @@
+{% set version = "1.4.2" %}
+{% set build_num = 0 %}
+
+package:
+  name: micromamba-server
+  version: {{ version }}
+
+source:
+  url: https://github.com/mamba-org/mamba/archive/refs/tags/micromamba-{{ version }}.tar.gz
+  sha256: dce034908d02d991c5e9aadeb9d01f139d027ba199aaeb1d47d543e3f24895d1
+
+build:
+  number: {{ build_num }}
+  string: {{ build_num }}
```

With this change, the bot will be able to auto-increment the build number.
```suggestion
  number: 0
  string: {{ PKG_BUILDNUM }}
```